recipes/gwsumm/meta.yaml

```
@@ -0,0 +1,105 @@
+{% set name = "gwsumm" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0dd3ec91cf51fe868e2f77065c15816c8e1ee87e4f0d3fdef03b64b340d7a19f
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - jsmin
+    - libsass
+    - pip
+    - python
+    - setuptools
+  run:
+    - python >=2.7
```

As this isn't `noarch` you shoudn't constrain the python version:
```suggestion
    - python
```